### ISSUE TITLE
IMDB Dataset Experiments

### ISSUE THREAD

**Issue by xkianteb** [eric-mitchell/direct-preference-optimization#27]
Are the IMDB experiments provided in the codebase? I do not see what  pre-trained sentiment classifier you used to collect additional data.

**Comment by rmrafailov** [comment]
The IMDB experiments use this sentiment model: "siebert/sentiment-roberta-large-english". You can construct the reward pipeline as:

reward_pipe = pipeline("text-classification", model="siebert/sentiment-roberta-large-english", device=0)
reward_kwargs = {"return_all_scores": True, "batch_size": 16, "truncation": True, "max_length": 512}

**Comment by eric-mitchell** [comment]
@xkianteb I don't expect we'll be able to get the complete code for the IMDb experiments released in the near future, but it's on our TODOs. We'll circle back on this issue when we have an update. Closing now, but feel free to re-open/follow up.

**Comment by DavidHerel** [comment]
Hi,

any updates for the IMDB experiments? It would be cool to replicate results from the paper.

**Comment by mohit-raghavendra** [comment]
Upvoting this, super interested in reproducing this particular task/dataset.
